src/modes/static.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(ctx) {
+  return {
+    stop: function() {
+      ctx.map.doubleClickZoom.enable();
```

Comment:
Actually, though, I'm confused as to why this needs to enable the zoom. Don't the other modes that disable zoom clean themselves up?